5.2.get_recipe_price.py

```diff
@@ -0,0 +1,14 @@
+def get_recipe_price(prices, optionals = None, **ingredients):
```

Comment:
When the function is a little bit complicated you need to have documentation or type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/